deployment/terraform/assets/cluster.tf

```diff
@@ -281,7 +281,7 @@ resource "aws_rds_cluster_instance" "cluster_instances" {
 }
 
 resource "aws_rds_cluster" "db_cluster" {
-  count               = var.app_instance_count > 0 ? 1 : 0
+  count               = var.app_instance_count > 0 && var.db_instance_engine != "cockroachdb" ? 1 : 0
```

Comment:
Same here.

Reply:
Actually, this is about `app_instance_count`, so I guess that is a typo or we can't solve that this way because we are going to run a cluster anyway.